zephyr/module.yml

```diff
@@ -0,0 +1,3 @@
+name: hal_ambiq
+build:
+  cmake: .
```

Comment:
Since you have a Kconfig option `CONFIG_AMBIQ_HAL`, it should be defined in `zephyr/modules/hal_ambiq/Kconfig` and you should have a
```
 kconfig-ext: True
```
here. See other HALs for examples.

Reply:
I changed it.